### PR TITLE
added qoute by Aleksandr and Wilhelm

### PR DESCRIPTION
**Pull request by mdjibril**
## ✅ New Quote Checklist
- [  ] The quote(s) added are in alphabetical order by author's first name and not ❌ duplicates.
- [  ] The quote(s) added are about **learning a foreign language**.
- [  ] The sentence attribute does not have inner quotation marks, and is ended with a period.
- [  ] Copy pasted the raw JSON `quotes.json` file to [JSONLint](https://jsonlint.com/) and made sure that the modified JSON file is valid.

### CODE REVIEW

**Review by kvharish — APPROVED** [review]

File: docs/how_to_contribute.md [hunk 1]
```diff
@@ -47,3 +47,52 @@ You can contribute in two ways, whichever is more convinient for you:
 ## 2️⃣ Via Local Machine
  
 🚧 Guide coming soon! 🔜
```
Comment (jayehernandez) [inline docs/how_to_contribute.md:49]: Hope you can remove this line already!

**Review by jayehernandez — COMMENTED** [review]
I totally forgot about updating the docs for the local machine way!

Thank you for that!! Hope you can update based on the comment and i'll merge after. :)

File: docs/how_to_contribute.md (hunk 1 above)
Comment (mdjibril) [reply docs/how_to_contribute.md]: Thank you. I have remove the line.

**Comment by mdjibril** [comment]
a **reminder** that #422  quotes has been approved by @kvharish  but yet to be merged. thank you for allowing me to contribute

**Comment by jayehernandez** [comment]
Thank you for contributing to Letra ☀️, @mdjibril!! Appreciate your contribution to this open source project. 🥳

---

Continue supporting Letra by:
- Leaving a star ⭐️ for this GitHub repository
- Downloading the free [Chrome extension](https://chrome.google.com/webstore/detail/letra/cjodkkjokggcaeacdhjliobekbnnmoio) and leaving a review
- Upvoting the [ProductHunt page](https://www.producthunt.com/posts/letra-2)

**Comment by jayehernandez** [comment]
@all-contributors please add @mdjibril for documentation

**Pull request merged**